### DISCUSSION TITLE
Is it possible to make "Image Tracking" without border?

### DISCUSSION THREAD

**Discussion by rifatwahid** [AR-js-org/AR.js#287] [Help] [answered]
Hello,
I want to use the simple "Image Tracking", I saw in the "[AR.js Marker generator](https://jeromeetienne.github.io/AR.js/three.js/examples/marker-training/examples/generator.html)" creates marker with border. I just need to create a marker without any border.

is it possible, please?

- Thanks in advance

**Comment by elfyboo** [comment]
You would have to use the NFT (Natural Feature Tracking) feature of AR.js, this allows you to just use an image as your marker. The image has to be sufficiently complex and non-symmetrical for best results

**Comment by elfyboo** [comment, marked as answer]
Check out 'Getting Started With Image Tracking' Header, it has information on how to create & use NFT's for this https://ar-js-org.github.io/AR.js-Docs/image-tracking/

**Reply by rifatwahid** [reply]
worked, thanks a lot ;)
